frontend/src/components/common/Filter/components/FilterContent/components/common/style.ts

```diff
@@ -0,0 +1,180 @@
+import styled from "@emotion/styled";
```

Comment:
Single source of truth for filtering styles! Broken down into sizeable and re-usable pieces!

Reply:
This is super cool! 👏